include/fmt/core.h

```diff
@@ -30,10 +30,10 @@
 # define FMT_HAS_INCLUDE(x) 0
 #endif
 
-#if defined(__GNUC__) && !defined(__clang__)
-# define FMT_GCC_VERSION (__GNUC__ * 100 + __GNUC_MINOR__)
```

Comment:
Please bring `FMT_GCC_VERSION` back as it is used in this header.

Reply:
It's also used in format.h before the inclusion of core.h, but sure.